--- liblastfm/src/CMakeLists.txt_old	2016-06-15 14:27:03.117114411 +0200
+++ liblastfm/src/CMakeLists.txt	2016-06-15 14:27:20.042073700 +0200
@@ -105,4 +105,4 @@

 file(GLOB liblastfm_HEADERS ${CMAKE_CURRENT_LIST_DIR}/*.h)
 list(APPEND liblastfm_HEADERS ${CMAKE_CURRENT_BINARY_DIR}/global.h)
-install(FILES ${liblastfm_HEADERS} DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/lastfm/)
+install(FILES ${liblastfm_HEADERS} DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/lastfm${LASTFM_LIB_VERSION_SUFFIX}/)
